pimd/pim_cmd.c

```diff
@@ -6433,11 +6432,10 @@ DEFUN_HIDDEN (interface_ip_pim_ssm,
 	return CMD_SUCCESS;
 }
 
-DEFUN (interface_ip_pim_sm,
+DEFUN_HIDDEN (interface_ip_pim_sm,
        interface_ip_pim_sm_cmd,
        "ip pim sm",
        IP_STR
-       PIM_STR
```

Comment:
IMO this is not required since the command had already been marked hidden. 
Also, this might cause mismatch in CLI tokens vs token description if made visible in the future.